src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,108 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final int VALID_AGE = 18;
+    private static final int INVALID_AGE = 17;
+    private static final int NEGATIVE_AGE = -5433;
+    private static final String DEFAULT_LOGIN = "defaultLogin";
+    private static final String VALID_PASSWORD = "defaultPassword";
+    private static final String INVALID_PASSWORD = "aaaaa";
+    private static RegistrationService registrationService;
+    private User testUser;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void beforeEach() {
+        testUser = new User();
+        testUser.setLogin(DEFAULT_LOGIN);
+        testUser.setPassword(VALID_PASSWORD);
+        testUser.setAge(VALID_AGE);
+    }
+
+    @AfterEach
+    void afterEach() {
+        Storage.people.clear();
+    }
+
+    @Test
+    void register_userIsNull_NotOk() {
```

Comment:
```suggestion
    void register_userIsNull_notOk() {
```
every part of the name of the test method has to start with a letter in lowerCase ( check it everywhere )

Reply:
Renamed in new commit. L8r suffixes: *_NotOk and *_Ok was named just according to Bohdan video.  ¯\_(ツ)_/¯
+corrected in new commit.